### PR TITLE
feat(packages): add search and user onboarding

### LINKED ISSUE #1: initial setup

### PR DESCRIPTION
**Pull request by Surbhi-sharma1**
add search and user onboarding libraries.
add sandbox examples 

GH-1

## Description

-add search and user onboarding in packages
- add sandbox examples

Fixes #1 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/combine-prs.yml
Comment (yeshamavani) [inline .github/workflows/combine-prs.yml:1]: is this file needed here ?

File: packages/search/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* fix(search-client): version update ([324ff0c](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/324ff0c)), closes [#0](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/0)
+* feat(search-client): add input templates for title and subtitle (#393) ([1163c12](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/1163c12)), closes [#393](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/393)
+
+
+
+
+
+## <small>1.1.9 (2021-10-27)</small>
+
+* fix(search-client): increase width of search dropdown and add ellipsis on overflow (#392) ([4d2f0af](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/4d2f0af)), closes [#392](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/392) [#0](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/0)
```
Comment (yeshamavani) [inline packages/search/CHANGELOG.md:52]: refernce to catalog

File: packages/search/projects/search-lib/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+
+
+
+
+## 5.4.0 (2023-07-28)
+
+* feat(user-onboarding): add element support and script for building (#1377) ([eebbe4e](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/eebbe4e)), closes [#1377](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1377) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352) [#1352](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/1352)
```
Comment (yeshamavani) [inline packages/search/projects/search-lib/CHANGELOG.md:16]: user-onboarding ???

File: packages/user-onboarding/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+
+
+
+## <small>1.0.1 (2021-10-24)</small>
+
+* fix: Tour not updating on route navigation (#358) ([d75ebfa](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/d75ebfa)), closes [#358](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/358)
+* fix(core): fix all version problems ([7847008](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/7847008)), closes [#0](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/0)
+* refactor(core): publishing user onboarding (#380) ([ecaf4c2](https://github.com/sourcefuse/loopback4-microservice-catalog/commit/ecaf4c2)), closes [#380](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/380) [#00](https://github.com/sourcefuse/loopback4-microservice-catalog/issues/00)
```
Comment (yeshamavani) [inline packages/user-onboarding/CHANGELOG.md:26]: same

File: .github/workflows/combine-prs.yml
Comment (Surbhi-sharma1) [reply .github/workflows/combine-prs.yml:1]: Removed

File: packages/search/CHANGELOG.md (hunk 1 above)
Comment (Surbhi-sharma1) [reply packages/search/CHANGELOG.md:52]: Done with all the changes

File: packages/user-onboarding/package.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    "build": "ng build",
+    "test": "ng test --no-watch",
+    "coverage": "ng test --code-coverage --no-watch",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
+    "userOnBoardingBuild": "ng build userOnboardingLib  && cp -r projects/user-onboarding-lib/src/assets projects/user-onboarding-lib/dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sourcefuse/loopback4-microservice-catalog.git",
```
Comment (yeshamavani) [inline packages/user-onboarding/package.json:20]: reference to catalog

File: packages/user-onboarding/projects/user-onboarding-element/CHANGELOG.md
Comment (yeshamavani) [inline packages/user-onboarding/projects/user-onboarding-element/CHANGELOG.md:1]: delete this file

File: packages/user-onboarding/projects/user-onboarding-lib/CHANGELOG.md
Comment (yeshamavani) [inline packages/user-onboarding/projects/user-onboarding-lib/CHANGELOG.md:1]: delete

File: packages/user-onboarding/projects/user-onboarding-lib/package.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  "ngPackage": {
+    "lib": {
+      "entryFile": "src/public-api.ts"
+    },
+    "dest": "dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sourcefuse/loopback4-microservice-catalog.git",
```
Comment (yeshamavani) [inline packages/user-onboarding/projects/user-onboarding-lib/package.json:50]: same

**Review by yeshamavani — CHANGES_REQUESTED** [review]

**Comment by yeshamavani** [comment]
also move the sandbox examples here

File: packages/search/projects/search-lib/CHANGELOG.md (hunk 2 above)
Comment (Surbhi-sharma1) [reply packages/search/projects/search-lib/CHANGELOG.md]: removed

File: packages/user-onboarding/CHANGELOG.md (hunk 3 above)
Comment (Surbhi-sharma1) [reply packages/user-onboarding/CHANGELOG.md]: removed

File: .github/workflows/docs.yml
[the remark this answers is not in the document]
Comment (Surbhi-sharma1) [reply .github/workflows/docs.yml:1]: removed

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -13,24 +15,27 @@ jobs:
           fetch-depth: 0
           token: ${{ secrets.RELEASE_COMMIT_GH_PAT }}
       - name: Setup Node
-        uses: actions/setup-node@v3.6.0
+        uses: actions/setup-node@v3
         with:
-          node-version: '16.x'
+          node-version: '18.x'
```
[the remark this answers is not in the document]
Comment (Surbhi-sharma1) [reply .github/workflows/release.yml:20]: changed to 20.

File: packages/search/projects/search-element/package.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "5.5.0",
+  "description": "A global search element for search microservice",
+  "main": "./search-element.js",
+  "private": false,
+  "scripts": {
+    "prepublishOnly": "cp package.json dist/package.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sourcefuse/loopback4-microservice-catalog.git",
```
[the remark this answers is not in the document]
Comment (Surbhi-sharma1) [reply packages/search/projects/search-element/package.json]: Done

File: .github/workflows/pull_request.yml
Comment (yeshamavani) [inline .github/workflows/pull_request.yml:1]: why did we remove this?

File: .github/workflows/release.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (yeshamavani) [reply .github/workflows/release.yml]: file deleted?

**Pull request closed without merging**